CONTRIBUTING.md

```diff
@@ -275,6 +275,14 @@ By giving feedback on this work (and related supporting work), you can help us m
 Finally, if nothing brings you more satisfaction than seeing every last issue labeled and all resolved issues closed, feel free to message @cart for a Bevy org role to help us keep things tidy.
 As discussed in [*How we're organized*](#how-were-organized), this role only requires good faith and a basic understanding of our development process.
 
+### How to adopt pull requests
+
+Occasionally authors of pull requests get busy or become unresponsive. This is a natural part of any open source project. To avoid blocking these efforts, these pull requests may be *adopted*, where another contributor creates a new pull request with the same content. If there is an old Pull request without the label that is without updates, comment to the organization whether it is appropriate to add the *[S-Adopt-Me](https://github.com/bevyengine/bevy/labels/S-Adopt-Me)* label, to indicate that it can be *adopted*.
+
+With this label added, it's best practice to fork the original author's branch. This ensures that they still get credit for working on it and that the commit history is retained. When the new pull request is ready, it should reference the original PR in the description. Then notify org members to close the original.
+
+* Example: `Adopted #number-original-pull-request`
```

Comment:
Nit: Not super required, but sometimes people at-mention the original author in their PR description to further give back some credit to that author. Is this something we should add here? Or is it mainly a preference thing that users can decide for themselves (and so we can exclude it from this document)?

Reply:
We put this part with examples just because in the description of issue (#5539) it asked for *"correct way to "adopt a PR" that also preserves credit of the original authors"*. Do you think this part may be deleted?